### ISSUE TITLE
Program does not run with any other than java version 8

### ISSUE THREAD

**Issue by fuggerjaki61** [fuggerjaki61/Search2SQL#1]
## Expected Behavior

Program should run with no exceptions thrown.

## Actual Behavior

Program starts and then throws error:

```
Exception in thread "main" org.reflections.ReflectionsException: Scanner TypeAnnotationsScanner was not configured
    at org.reflections.Store.get(Store.java:39)
    at org.reflections.Store.get(Store.java:61)
    at org.reflections.Store.get(Store.java:46)
    at org.reflections.Reflections.getTypesAnnotatedWith(Reflections.java:429)
```

Input line appears and query can be entered. If query is entered, program throws error again and terminates.

## Steps to Reproduce the Problem

Just run the program on a machine with java 9 or higher.

## Problem

Reflections library does not work with java 9+. See [StackOverflow question](https://stackoverflow.com/questions/60559501/scanner-typeannotationsscanner-was-not-configured-when-used-from-a-referenced-l).

## Specifications

  - Version: 0.0.1
  - Platform: Oracle Java 15

**Comment by fuggerjaki61** [comment]
Possible solution described by the Stackoverflow mentions downgrading to 0.9.10 (or lower) or using https://github.com/aschoerk/reflections8 (Reflections API for java 8). 

Downgrading to 0.9.10 (or lower) does not work. Using `reflections8` fixes the problem.

**Comment by fuggerjaki61** [comment]
Fixed this issue with commit 0a34ab3655018d70db5f2c5a7f535cd3afe3a7ab. Will change library when bug is fixed.
